### PR TITLE
Chore: Documentation and input / output cols

### PR DESCRIPTION
**Pull request by elboy3**
Small cleanup leading up to S2S sprint during onsite week

### CODE REVIEW

File: dataquality/loggers/model_logger/seq2seq.py [hunk 1]
```diff
@@ -135,7 +154,7 @@ def _get_data_dict(self) -> Dict:
             C.epoch.value: [self.epoch] * batch_size,
         }
         if self.split == Split.inference:
-            data["inference_name"] = [self.inference_name] * batch_size
+            data[C.inference_name.value] = [self.inference_name] * batch_size
```
Comment (setu4993) [inline dataquality/loggers/model_logger/seq2seq.py:157]: Do we need a `.value` here? I think it works simply, too?
```suggestion
            data[C.inference_name] = [self.inference_name] * batch_size
```

File: dataquality/schemas/seq2seq.py [hunk 2]
```diff
@@ -3,18 +3,23 @@
...
+
+
+class Seq2SeqOutputCols(str, Enum):
+    id = "id"
     token_deps = "token_deps"
     token_gold_probs = "token_gold_probs"
     # Mypy complained about split as an attribute, so we use `split_`
     split_ = "split"
     epoch = "epoch"
+    inference_name = "inference_name"
```
Comment (setu4993) [inline dataquality/schemas/seq2seq.py:22]: Why `inference_name`?

**Review by setu4993 — APPROVED** [review]

File: dataquality/schemas/seq2seq.py (hunk 2 above)
Comment (elboy3) [reply dataquality/schemas/seq2seq.py:22]: For line 157 in dataquality/loggers/model_logger/seq2seq.py

File: dataquality/loggers/model_logger/seq2seq.py (hunk 1 above)
Comment (elboy3) [reply dataquality/loggers/model_logger/seq2seq.py:157]: I did value for consistency with the above 5 lines, but I can do a test removing .value from all of them and push that up

Comment (elboy3) [reply dataquality/loggers/model_logger/seq2seq.py:157]: Yes we do! This creates a dict that later gets cast to a vaex DataFrame, which can't have enum as the keys to the dict 

The following errors:
```python
from enum import Enum
import vaex

class MyType(str, Enum):
    a = "a"
    b = "b"
    c = "c"

df = vaex.from_dict(
    {
        MyType.a: [1, 2, 3],
        MyType.b: [4, 5, 6],
        MyType.c: [7, 8, 9],
    }
)
```

Comment (setu4993) [reply dataquality/loggers/model_logger/seq2seq.py:157]: TIL!

File: dataquality/schemas/seq2seq.py (hunk 2 above)
Comment (setu4993) [reply dataquality/schemas/seq2seq.py:22]: My question was why call it inference _name_? _Name_ as a column name is too generic and doesn't feel like that's what we should be getting?

**Review by anthonycorletti — APPROVED** [review]

**Pull request merged**